plugins/module_utils/scaleway.py

```diff
@@ -12,7 +12,7 @@
 
 def scaleway_argument_spec():
     return dict(
-        api_token=dict(required=True, fallback=(env_fallback, ['SCW_TOKEN', 'SCW_API_KEY', 'SCW_OAUTH_TOKEN', 'SCW_API_TOKEN']),
+        api_token=dict(fallback=(env_fallback, ['SCW_TOKEN', 'SCW_API_KEY', 'SCW_OAUTH_TOKEN', 'SCW_API_TOKEN']),
```

Comment:
Do not remove `required=True` here. Instead, fix the docs fragment to have `required: true`.

Also, remove the changelog fragment. This should be a docs-only change.

Reply:
Hmmm, I took an example from `alicloud`, `online` (same company as Scaleway), `postgres` (with user/pass), `vexeta` and `xenserver` : none of them as an api_token or user/pass required if an environment variable is used. Maybe we should harmonize all credentials variables? Or maybe I don't understand the docs 😅

Reply:
I don't think a password is required for `postgres`. For the others, I guess they did the same error as `scaleway` and actually require it, but do not document it as required. Which is wrong.

Also, to be clear: `api_token` **is** required for scaleway. You cannot use the modules without `api_token`. It is **not** required to pass it as a module option though, you can also pass it as an environment variable. But you still need to pass it one way or the other.

This must **not** be changed, otherwise you break the modules. What has to be fixed is the documentation, which currently is wrong.

Reply:
Ok thanks for your explanation 😄  (It's my first Ansible (and my first Python) module).
I'll make the changes.